docs/guides/deploy/local.md

```
@@ -0,0 +1,184 @@
+docker compose up -d
+```
+
+Verify it is running:
+
+```bash
+docker logs local-da
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)